modules/check_in/spec/services/travel_claim/client_spec.rb

```diff
@@ -123,11 +123,12 @@
         allow_any_instance_of(Faraday::Connection).to receive(:post).with(anything).and_raise(exception)
       end
 
-      it 'logs message and raises exception' do
+      it 'logs message and returns original error' do
         expect_any_instance_of(SentryLogging).to receive(:log_message_to_sentry)
-        expect do
-          subject.submit_claim(token: access_token, patient_icn: icn, appointment_date: appt_date)
-        end.to raise_exception(exception)
```

Comment:
Pls remove `exception` defined as Common::Exceptions::BackendServiceException in all the tests

Reply:
why? that's what's being raised by the connection to simulate the downstream error response.

Reply:
Got it, missed the usage in downstream error part